docs/core/diagnostics/metrics-collection.md

```diff
@@ -63,10 +63,10 @@ Press p to pause, r to resume, q to quit.
     hats-sold (Count / 1 sec)                          4
 ```
 
-`dotnet-counters` can be run with a different set of metrics to see some of the built-in instrumentation from the .NET runtime:
+`dotnet-counters` can be run with a different set of metrics to see some of the built-in instrumentation from the .NET `System.Runtime`:
```

Comment:
```suggestion
`dotnet-counters` can be run with a different set of metrics to see some of the built-in instrumentation from the .NET runtime:
```

Reply:
I prefer the original wording here. I've never seen it referred to as "the .NET System.Runtime".